sqlglot/expressions.py

```diff
@@ -191,6 +191,13 @@ def alias(self) -> str:
             return self.args["alias"].name
         return self.text("alias")
 
+    @property
+    def alias_column_names(self) -> t.List[str]:
+        table_alias = self.args.get("alias")
+        if not table_alias:
+            return []
+        return [c.name for c in table_alias.args.get("columns") or []]
+
```

Comment:
Moved here because it wasn't available to `Table`, although it can also apply to it. Since we have some other helpers like `alias` in `Expression`, this change felt ok.